test/periphery/PixCashierUpgradeable.test.ts

```diff
@@ -141,21 +155,27 @@ describe("Contract 'PixCashierUpgradeable'", async () => {
       await proveTx(pixCashier.setPauser(deployer.address));
       await proveTx(pixCashier.setPauser(deployer.address));
       await proveTx(pixCashier.pause());
-      await expect(pixCashier.connect(cashierClient).cashOutConfirm(tokenAmount))
+      await expect(pixCashier.connect(cashierClient).cashOutConfirm(tokenAmount, TRANSACTION_ID))
         .to.be.revertedWith(REVERT_MESSAGE_IF_CONTRACT_IS_PAUSED);
     })
 
     it("Is reverted if the user's cash-out balance has not enough tokens", async () => {
-      await proveTx(pixCashier.connect(cashierClient).cashOut(tokenAmount - 1));
-      await expect(pixCashier.connect(cashierClient).cashOutConfirm(tokenAmount))
+      await proveTx(pixCashier.connect(cashierClient).cashOut(tokenAmount - 1, TRANSACTION_ID));
```

Comment:
Codacy has a fix for the issue: [Replace `pixCashier.connect(cashierClient).cashOut(tokenAmount·-·1,·TRANSACTION_ID)` with `⏎········pixCashier⏎··········.connect(cashierClient)⏎··········.cashOut(tokenAmount·-·1,·TRANSACTION_ID)⏎······`](https://app.codacy.com/gh/cloudwalk/brlc-token/pullRequest?prid=9795032)
```suggestion
      await proveTx(
```